src/js/tracker.js

```diff
@@ -1331,6 +1331,14 @@
 				geolocationContextAdded = true;
 				navigatorAlias.geolocation.getCurrentPosition(function (position) {
 					var coords = position.coords;
+
+					// we don't use Number.isInteger here as it is not supported in IE11 and below
+					if (position.timestamp % 1 !== 0) {
```

Comment:
For the 1/1000 times you're exactly on a second this will do the wrong thing because 0 === 0.00000

Reply:
Probably better to try `if (position.timestamp && position.timestamp.toString().indexOf('.') > -1)`

Reply:
The standard is pretty clear though, this is meant to be a long long int with millisecond precision; are you actually seeing this in the wild as a float?